docs/configuring.rst

```diff
@@ -101,6 +102,14 @@ Miscellaneous options
     If specified, Google Analytics' javascript is included in your pages.
     Set the value to the ID provided to you by google (like ``UA-XXXXXXX``).
 
+.. confval:: analythics_anonymize_ip
+
+   :type: boolean
+   :default: ``False``
+
+   Enabling this will make it so the users’ IP addresses are anonymized
```

Comment:
This can be simplified, e.g. "Anonymize visitor IP addresses in Google Analytics."